docker/Dockerfile

```diff
@@ -114,7 +60,7 @@ ENV RUSTUP_HOME=/opt/rustup \
     PATH=/opt/cargo/bin:$PATH
 
 # Make sure /rust-toolchain.toml has the same version
-ARG rust_version=1.66.1
+ARG rust_version=1.79.0
```

Comment:
could we get this from rust config instead?

Reply:
yep!